src/Dflydev/FigCookies/Modifier/SameSite.php

```diff
@@ -24,25 +26,25 @@ private function __construct(string $value)
         $this->value = $value;
     }
 
-    public static function strict() : self
+    public static function strict(): self
     {
-        return new self(self::STRICT);
+        return new static(self::STRICT);
     }
 
-    public static function lax() : self
+    public static function lax(): self
     {
-        return new self(self::LAX);
+        return new static(self::LAX);
     }
 
-    public static function none() : self
+    public static function none(): self
     {
-        return new self(self::NONE);
+        return new static(self::NONE);
```

Comment:
it's a final class, static is useless here

Reply:
But is it wrong?